sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 4.6.0-beta.1 (Unreleased)
+## 4.6.0 (2021-08-10)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added `ContainerServiceNewKubernetesVersionAvailableEventData` system event.
```

Comment:
That's a mouthful...

Reply:
It sure is, but it looks like it is following the established patterns.